pyEpiabm/pyEpiabm/output/_csv_writer.py

```diff
@@ -4,13 +4,14 @@
 
 import csv
 import typing
+import os
 
 from pyEpiabm.output.abstract_reporter import AbstractReporter
 
 
 class _CsvWriter(AbstractReporter):
     def __init__(self, folder: str, filename: str, fieldnames: typing.List,
-                 clear_folder: bool = True):
+                 clear_folder: bool = False):
```

Comment:
If we are changing the default behaviour, can you check this still works with the workflows? In particular this will affect the process of gif generation (which is based on all png files it can find in the output folder, and so may mistakenly include historic images if they are left in the folder - probably best to explicitly clear the folder in this workflow), but also worth checking it is happy to overwrite output files of the same name as well

Reply:
Doesn't change functionality of 1st simulation run.
Subsequent simulation runs will now overwrite individual output files instead of deleting and recreating the output folder.

Reply:
Makes sense, the only change I can think of is with the pngs so maybe best to at least document this - its only an issue if you also decrease the number of timesteps then rerun the plotter as it won't overwite images from the old simulation I think?